VCFv4.3.tex

```diff
@@ -503,6 +507,10 @@ \subsubsection{Genotype fields}
   All phased genotypes that do not contain a PS subfield are assumed to belong to the same phased set.
   If the genotype in the GT field is unphased, the corresponding PS field is ignored.
   The recommended convention is to use the position of the first variant in the set as the PS identifier (although this is not required).
+  \item PQL (List of integers): The list of PQs one for each phase set in PSL (encoded like PQ)
+  \item PSL (List of non-negative 32-bit Integer): The list of PSs one for each pipe ($\mid$) in the GT field, specifying the phase set for the allele prior to the pipe.
+  A given sample-genotype must not have values for both PS and PSL. 
+  However, they are interoperable, in that a PS mentioned in one variant can be references in a PSL in another.
```

Comment:
They are not interoperable and can't be mixed. Take the following example:
```
contig	1	snp1	A	T	.	.		GT:PS	0|1:1
contig	2	snp2	A	G	.	.		GT:PSL	0/1|:1
```
The G at snp2 is phased with snp1, but we don't know if it's phased with the A or the T allele since PS applies to the set of all alleles whereas PSL applies to just an individual one.

Reply:
I agree with the conclusion that PS and PSL are not interoperable and can't be mixed. However, I believe the example is incorrect in showing PSL having a single value. According to the proposed definition, it must have two values.